.eslintrc

```diff
@@ -24,6 +24,9 @@
             "allowTemplateLiterals": false
         }],
         "brace-style": 2,
+        "keyword-spacing": ["error", {
```

Comment:
I've edited the eslint config to catch the keyword spacing issues (see [keyword-spacing](https://eslint.org/docs/latest/rules/keyword-spacing)). I hope that's alright.

Reply:
Ah thanks jndeed it is, it would be great if you added this to https://github.com/TinyWebEx/AddonTemplate, too, as that is my template for all add-ons.